DatabaseService/PostgreSQL/docker_context/create_tables.sql

```diff
@@ -1,13 +1,13 @@
 rollback;
 
 create table if not exists users (
-    user_id varchar(255) primary key
+    user_id varchar(12) primary key
```

Comment:
why have we cut the size of this down? there could definitely be cases of longer then 12 character user id's

Reply:
I saw in one of the schemas that we were limiting this to 12 at most hence the decision to change this. Is that not the case? I can change it back to 255 though or some other number that you think would be better.

Reply:
Ya I think Mitch may have done that once but was a similar concern and got reverted. Sorry if it was missed. If possible I think we put it back to 255

Reply:
Reverted to 255. Resolving comment.